install problem with 2.1.0.008-DSM6

this may also be related to DSM 6.1.3-15152 Update 3, but I am not sure.

I tried to install 2.2.0.006-SNAPSHOT-WIP, but got errors like
```
2017-08-17 22:03:22.305 [WARN ] [org.jline] - Unable to create a system terminal, creating a dumb terminal (enable debug logging for more information)
2017-08-17 22:03:22.502 [WARN ] [org.jline] - Unable to retrieve infocmp for type vt102
java.io.IOException: Cannot run program "infocmp": error=2, No such file or directory
(snip)
Caused by: java.io.IOException: error=2, No such file or directory
(snip)
```

Then I tried to go back to 2.1.0.008-DSM6 (prior version was .0004). Now it does not recognize my conf and addons file on /volume1/public/openHAB2 anymore:

My @appstore directory looks good after installation of the spk:
```
admin@DiskStation:/volume1/@appstore/openHAB2$ ls -al
total 56
drwxr-xr-x  4 openhab2 root   4096 Aug 20 12:44 .
drwxrwxrwx 22 root     root   4096 Aug 20 12:43 ..
lrwxrwxrwx  1 openhab2 root     31 Aug 20 12:44 addons -> /volume1/public/openHAB2/addons
lrwxrwxrwx  1 openhab2 root     29 Aug 20 12:44 conf -> /volume1/public/openHAB2/conf
-rwxrwxrwx  1 openhab2 users 11232 Jun 27 23:14 LICENSE.TXT
drwxrwxrwx  6 openhab2 users  4096 Aug 20 12:44 runtime
-rwxrwxrwx  1 openhab2 users   121 Jan 20  2017 start.bat
-rwxrwxrwx  1 openhab2 users    72 Jan 20  2017 start_debug.bat
-rwxr-xr-x  1 openhab2 root   1132 Nov 17  2013 start_debug_runtime.sh
-rwxrwxrwx  1 openhab2 users    73 Jan 20  2017 start_debug.sh
-rwxr-xr-x  1 openhab2 root    930 Nov 17  2013 start_runtime.sh
-rwxrwxrwx  1 openhab2 users   116 Jan 20  2017 start.sh
-rwxr-xr-x  1 openhab2 root     65 Nov 17  2013 stop_runtime.sh
drwxrwxrwx  5 openhab2 users  4096 Aug 20 12:44 userdata
```
but the conf and addons folders are not recognized:
```
admin@DiskStation:/volume1/@appstore/openHAB2/userdata/logs$ cat openhab.log
2017-08-20 12:46:18.308 [WARN ] [org.apache.felix.fileinstall        ] - /volume1/@appstore/openHAB2/addons does not exist, please create it.
2017-08-20 12:46:18.356 [WARN ] [org.apache.felix.fileinstall        ] - Root path does not exist: /volume1/@appstore/openHAB2/addons
2017-08-20 12:46:53.966 [INFO ] [.dashboard.internal.DashboardService] - Started dashboard at http://192.168.178.79:8080
2017-08-20 12:46:53.972 [INFO ] [.dashboard.internal.DashboardService] - Started dashboard at https://192.168.178.79:8443
2017-08-20 12:47:02.670 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'things' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:02.674 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'scripts' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:02.678 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'items' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:02.681 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'persistence' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:02.684 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'sitemaps' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:02.686 [WARN ] [.core.internal.folder.FolderObserver] - Directory 'rules' does not exist in '/volume1/@appstore/openHAB2/conf'. Please check your configuration settings!
2017-08-20 12:47:07.440 [WARN ] [thome.ui.internal.chart.ChartServlet] - Illegal argument in chart: No Persistence service found.
```
As a workaround, I `mount --bind` my directories. This works well




<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/48453234-install-problem-with-2-1-0-008-dsm6?utm_campaign=plugin&utm_content=tracker%2F19715072&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F19715072&utm_medium=issues&utm_source=github).
</bountysource-plugin>

Hello

I wanted to start the file start.sh in manual
and I have this as error message

```
Launching the openHAB runtime ...
Unable to update instance pid: / volume1 / @ appstore / openHAB2 / userdata / tmp / instance s / instance.properties (Permission denied)
java.lang.RuntimeException: /volume1/@appstore/openHAB2/userdata/logs/openhab.lo g (Permission denied)
        at org.apache.karaf.main.util.BootstrapLogManager.getDefaultHandlerInter nal (BootstrapLogManager.java:102)
        at org.apache.karaf.main.util.BootstrapLogManager.getDefaultHandlersInte rnal (BootstrapLogManager.java:137)
        at org.apache.karaf.main.util.BootstrapLogManager.getDefaultHandlers (Boo tstrapLogManager.java:70)
        at org.apache.karaf.main.util.BootstrapLogManager.configureLogger (Bootst rapLogManager.java:75)
        at org.apache.karaf.main.Main.launch (Main.java:244)
        at org.apache.karaf.main.Main.main (Main.java:178)
Caused by: java.io.FileNotFoundException: / volume1 / @ appstore / openHAB2 / userdata / l ogs / openhab.log (Permission denied)
        at java.io.FileOutputStream.open0 (Native Method)
        at java.io.FileOutputStream.open (FileOutputStream.java:270)
        at java.io.FileOutputStream. <init> (FileOutputStream.java:213)
        at org.apache.karaf.main.util.BootstrapLogManager $ SimpleFileHandler.open (BootstrapLogManager.java:193)
        at org.apache.karaf.main.util.BootstrapLogManager $ SimpleFileHandler. <ini t> (BootstrapLogManager.java:182)
        at org.apache.karaf.main.util.BootstrapLogManager.getDefaultHandlerInter nal (BootstrapLogManager.java:100)
        ... more
SESSION 2017-09-04 17: 27: 08.668 -------------------------------------- ---------
eclipse.buildId = unknown
java.version = 1.8.0_144
java.vendor = Oracle Corporation
BootLoader constants: OS = linux, ARCH = arm, WS = gtk, NL = en_US

! ENTRY org.eclipse.osgi 4 0 2017-09-04 17: 27: 08.675
MESSAGE Error reading configuration: / volume1 / @ appstore / openHAB2 / userdata / cache /org.eclipse.osgi/.manager/.fileTableLock (Permission denied)
! STACK 0
java.io.FileNotFoundException: /volume1/@appstore/openHAB2/userdata/cache/org.ec lipse.osgi / .manager / .fileTableLock (Permission denied)
        at java.io.RandomAccessFile.open0 (Native Method)
        at java.io.RandomAccessFile.open (RandomAccessFile.java:316)
        at java.io.RandomAccessFile. <init> (RandomAccessFile.java:243)
        at org.eclipse.osgi.internal.location.Locker_JavaNio.lock (Locker_JavaNio .java: 36)
        at org.eclipse.osgi.storagemanager.StorageManager.lock (StorageManager.ja goes: 388)
        at org.eclipse.osgi.storagemanager.StorageManager.open (StorageManager.ja goes: 701)
        at org.eclipse.osgi.storage.Storage.getChildStorageManager (Storage.java: 1792)
        at org.eclipse.osgi.storage.Storage.getInfoInputStream (Storage.java:1809)
        at org.eclipse.osgi.storage.Storage. <init> (Storage.java:129)
        at org.eclipse.osgi.storage.Storage.createStorage (Storage.java:88)
        at org.eclipse.osgi.internal.framework.EquinoxContainer. <init> (EquinoxCo ntainer.java:66)
        at org.eclipse.osgi.launch.Equinox. <init> (Equinox.java:31)
        at org.eclipse.osgi.launch.EquinoxFactory.newFramework (EquinoxFactory.ja goes: 24)
        at org.apache.karaf.main.Main.launch (Main.java:256)
        at org.apache.karaf.main.Main.main (Main.java:178)
```

--- a/scripts/start-stop-status.sh
+++ b/scripts/start-stop-status.sh
@@ -5,7 +5,7 @@
 
 DAEMON_USER="`echo ${SYNOPKG_PKGNAME} | awk {'print tolower($_)'}`"
 DAEMON_ID="${SYNOPKG_PKGNAME} daemon user"
-ENGINE_SCRIPT="start.sh"
+ENGINE_SCRIPT="start.sh daemon"
 DAEMON_USER_SHORT=`echo ${DAEMON_USER} | cut -c 1-8`
 PIDFILE="/var/services/homes/${DAEMON_USER}/.daemon.pid"
 

